frontends/qt/main.cpp

```diff
@@ -32,6 +32,35 @@ static WebClass* webClass;
 static QMutex webClassMutex;
 static QSystemTrayIcon* trayIcon;
 
+class BitBoxApp : public QApplication
+{
+public:
+    BitBoxApp(int &argc, char **argv): QApplication(argc, argv)
+    {
+    }
+
+#if defined(Q_OS_MACOS)
+    bool event(QEvent *event) override
+    {
+        if (event->type() == QEvent::FileOpen) {
+            QFileOpenEvent* openEvent = static_cast<QFileOpenEvent*>(event);
+            if (!openEvent->url().isEmpty()) {
+                // This is only supported on macOS and is used to handle URIs that are opened with
+                // the BitBoxApp, such as "aopp:..." links. The event is received and handled both
+                // if the BitBoxApp is launched and also when it is already running, in which case
+                // it is brought to the foreground automatically.
+
+                // TODO: handle the URI.
+                // After removing the messagebox here, remove the QMessageBox include above.
+                QMessageBox::information(NULL, QString("Handle URI"), openEvent->url().toString());
```

Comment:
How about something like this instead:

```cpp
std::cerr << "Handle URI: " << openEvent->url().toString() << std::endl
```

- no need for qmessagebox
- might even keep it in the release; we already log blocked `systemOpen` URLs in this way
- feels much less distruptive even if it's in a staging branch